src/components/pages/Discover.tsx

```diff
@@ -0,0 +1,179 @@
+/**
+ * The discover tab is where members can learn more about
+ * systemic inequality and how solutions like UBI can
+ * help prevent it! More information about Raha and
+ * partnerships can be pushed here.
+ */
+import * as React from "react";
+import {
+  View,
+  Text,
+  TouchableHighlight,
+  WebView as WebViewNative,
+  TextProps,
+  ScrollView,
+  Linking
+} from "react-native";
+
+import { MemberSearchBar } from "../shared/MemberSearchBar";
+import { NavigationScreenProp, withNavigation } from "react-navigation";
+import { RouteName } from "../shared/Navigation";
+import { SafeAreaView } from "../../shared/SafeAreaView";
+
+export const DiscoverWebView: React.StatelessComponent = ({
+  navigation
+}: any) => {
+  return <WebViewNative source={{ uri: navigation.getParam("uri") }} />;
+};
+
+type DiscoverCardRaw = {
+  header?: string | string[];
+  title: string | string[];
+  footer?: string | string[];
+  uri: string;
```

Comment:
this seems a little strict for now since not every card will necessarily be pointing to a remote source, but ok